lang/semgrep-grammars/src/semgrep-kotlin/grammar.js

```diff
@@ -31,12 +31,17 @@ module.exports = grammar(standard_grammar, {
             )
         },
 
+				typed_metavar: $ =>  seq(
+					"(", $.simple_identifier, ":", $._type, ")"
+				), 
+
         // Statement ellipsis: '...' not followed by ';'
         _expression: ($, previous) => {
             return choice(
                 previous,
                 $.ellipsis,  // statement ellipsis
-                $.deep_ellipsis
+                $.deep_ellipsis,
+							  $.typed_metavar
```

Comment:
please fix the indentation. Probably because you use TAB. Avoid using TABS, use regular space (you can usually configure your editor for that).

Reply:
whoops, i didnt have `expandtab` in my vimrc, thanks